elixir/apps/domain/lib/domain/resources.ex

```diff
@@ -168,6 +171,16 @@ defmodule Domain.Resources do
     end
   end
 
+  def create_internet_resource(%Accounts.Account{} = account) do
+    attrs = %{type: :internet, name: "Internet"}
+    changeset = Resource.Changeset.create(account, attrs)
+
+    with {:ok, resource} <- Repo.insert(changeset) do
+      :ok = broadcast_resource_events(:create, resource)
+      {:ok, resource}
+    end
+  end
```

Comment:
Would it make sense for this function to use an upsert like functionality rather than a plain insert?  Was just thinking that if this ever got mistakenly called twice we'd end up with 2 internet resources for an account right?

Reply:
Added an index for prevent creating duplicate ones.